homework/index.js

```diff
@@ -30,18 +30,94 @@
     return elem;
   }
 
+  function render(parent, repositories) {
```

Comment:
This may have the name of a singuler instead of a plural form while it's an object of a one repository, what do you say? :)